MyBooking/Controllers/AdvertController.cs

```diff
@@ -44,8 +44,8 @@ public ActionResult<AdvertModel> GetByAddress(string advertAddress)
             return new AdvertModel(advert);
         }
 
-        /// GET Api/Advert/ByFilters/{animals,countPeople,minPrice,maxPrice}
-        [HttpGet("ByFilters{filtersModel}")]
+        /// GET Api/Advert/ByFilters
+        [HttpGet("ByFilters")]
```

Comment:
Will it take params for GET response?

Reply:
Yes, if I’m not mistaken, because an object is transferred that can be serialized automatically.
But also, I redid the transfer method